plugins/MobileMessaging/Model.php

```diff
@@ -233,7 +233,7 @@ public function getCredentialManagerSettings(): array
     public function getDelegatedManagement(): bool
     {
         $option = Option::get(MobileMessaging::DELEGATED_MANAGEMENT_OPTION);
-        return true === $option || 'true' === $option || 1 === $option;
+        return true === $option || 'true' === $option || 1 === $option || '1' === $option;
```

Comment:
It would be nice to have `Option::isTruthy('option name')` method that would do this.
Or if we can't accept strings as truthy, then something is `Option::isOn` (and possibly a counterpart `Option::isOff`). What do you think?